src/frontend/next/src/pages/index.tsx

```diff
@@ -1,60 +1,25 @@
+import { FC } from 'react';
 import Head from 'next/head';
-import styles from '../styles/Home.module.css';
```

Comment:
It looks like we haven't got CSS figured out yet, is that happening in a follow-up?

Reply:
File: [telescope/src/frontend/gatsby/src/pages/index.js](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/gatsby/src/pages/index.js)

```js
import React from 'react';

import PageBase from './PageBase';
import Banner from '../components/Banner';
import Posts from '../components/Posts';
import BackToTopButton from '../components/BackToTopButton';

export default function IndexPage() {
  return (
    <PageBase title="Home">
      <Banner />
      <BackToTopButton />
      <main className="main">
        <Posts />
      </main>
    </PageBase>
  );
}
```

We don't have a CSS in this component. The `PageBase` component holds the global `CSS` and `SEO`. 
With Next.js, we may have to break the `PageBase`. Reading this [Next.js doc](https://nextjs.org/docs/basic-features/built-in-css-support) I understood that the "global CSS" would be defined on the file: [next/src/pages/_app.tsx](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/next/src/pages/_app.tsx). But I would appreciate feedback on this approach.  

Note about `SEO`: I'm not sure if we could add our `SEO` to `_app.tsx` or if we need to add it to each page.

Reply:
@PedroFonsecaDEV : We can defined global CSS in MUI theme as well. I think that would be a better approach to have global styling in the theme.

Reply:
For global styling, the work happening in #1549 is probably the right approach.